cst438Assignment2/src/main/java/cst438hw2/domain/City.java

```diff
@@ -0,0 +1,129 @@
+package cst438hw2.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+@Entity
+@Table(name="city")
+public class City
+{
+
+   @Id
+   private int ID;
+   private String name;
+   private String district;
+   private int population;
+   
+   // Transient marks extra field in an Entity class
+   //  that is not read/written to the database.
+   
+   //@Transient TimeAndTemp weather;
+    
+   
+   @ManyToOne
+   @JoinColumn(name="countrycode", referencedColumnName="code")
+   private Country country;
+   
+   public City() { }
+   
+   public City(int ID, String name, String district, int population, Country country) {
+      this.ID = ID;
+      this.name = name;
+      this.district = district;
+      this.population = population;
+      this.country = country;
+   }
+
+   public int getID() {
+      return ID;
+   }
+
+   public void setID(int iD) {
+      ID = iD;
```

Comment:
To keep naming conventions consistent, I would keep iD as ID and to make things clearer, one thing I did was do this.variable_name = variable_name so I don't have to change the second variable name and so it's more clear what you're trying to do.